tests/callbacks_test.cpp

```diff
@@ -152,6 +152,31 @@ void CallbacksFullFunctionTest(OptimizerType& optimizer,
   REQUIRE(cb.calledStepTaken == calledStepTaken);
 }
 
+template<typename OptimizerType>
+void EarlyStopCallbacksFunctionTest(OptimizerType& optimizer)
+{
+  arma::mat data, testData, shuffledData;
+  arma::Row<size_t> responses, testResponses, shuffledResponses;
+
+  LogisticRegressionTestData(data, testData, shuffledData,
+      responses, testResponses, shuffledResponses);
+  
+  LogisticRegression<> lr(shuffledData, shuffledResponses, 0.5);
+  arma::mat coordinates = lr.GetInitialPoint();
+  LogisticRegressionValidationFunction lrValidation(lr, coordinates);
+
+  EarlyStopAtMinLoss<LogisticRegressionValidationFunction,
+  arma::mat, arma::Row<size_t>> cb(
+    lrValidation, testData, testResponses);
+
+  optimizer.Optimize(lr, coordinates, cb);
+}
+
+TEST_CASE("EarlyStopAtMinLossCallbackTest", "[CallbacksTest]")
+{
+  SMORMS3 smorms3;
```

Comment:
@zoq  I have added the function as you have proposed, but I am not sure that we are stopping the optimization as we were doing before